releasetool/data/roles/download_rpms/tasks/main.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- name: fetch tag content from koji
+  include_tasks: download.yaml
+  loop: "{{ oses }}"
```

Comment:
This one is a bit of an odd variable as it reads a bit weird for me. As verbose as it may be length wise, I tend to prefer operating_systems here. I did sit here for a few seconds wondering what oses were.